src/map/mob.c

```diff
@@ -3733,74 +4010,126 @@ bool mob_parse_dbrow(char** str) {
 #endif
 
 	//Tests showed that chase range is effectively 2 cells larger than expected [Playtester]
-	if (db->range3 > 0)
-		db->range3 += 2;
+	if (entry->range3 > 0)
```

Comment:
If fix this code, will be no more range3 equal to 1 or 2. May be better remove this code?